plugins/bigquery/dbt/adapters/bigquery/impl.py

```diff
@@ -551,10 +553,10 @@ def _partitions_match(
         if not is_partitioned and not conf_partition:
             return True
         elif conf_partition and table.time_partitioning is not None:
-            table_field = table.time_partitioning.field
-            table_granularity = table.partitioning_type
-            return table_field == conf_partition.field \
-                and table_granularity == conf_partition.granularity
+            table_field = table.time_partitioning.field.lower()
+            table_granularity = table.partitioning_type.lower()
```

Comment:
do you think `table.partitioning_type` could ever be `None` here? I'd think `table.time_partitioning is not None` also means `table.partitioning_type` has a value, just being nit-picky

Reply:
I think this is fine, you can ignore! I dug around docs a little and I think
> table.time_partitioning is not None also means table.partitioning_type has a value

holds true

Reply:
ah good catch though! if one of our intrepid `dbt-bigquery` beta testers manages to get an error like `NoneType has no attribute lower`, then we'll know who to blame (me) and where to fix it (here)